identity-composer-component.js

```diff
@@ -144,6 +144,13 @@ function Ctrl($q, brCredentialLibraryService, brMediaQueryService) {
       });
   };
 
+  self.detail = function(credential, showCredential, $event) {
```

Comment:
Naming convention for functions called in response to an event/action is: `onFoo`. So this should be something like: `onInfoClicked` or `onShowDetails` or similar.